src/sql/parts/profiler/contrib/profilerActions.contribution.ts

```diff
@@ -30,15 +34,45 @@ const newProfilerSchema: IJSONSchema = {
 
 CommandsRegistry.registerCommand({
 	id: 'profiler.newProfiler',
-	handler: (accessor: ServicesAccessor) => {
-		let editorService: IEditorService = accessor.get(IEditorService);
+	handler: (accessor: ServicesAccessor, ...args: any[]) => {
+		let connectionProfile: ConnectionProfile = undefined;
```

Comment:
should this be IConnectionProfile?

Reply:
@abist If I use IConectionProfile, I will have to convert it to ConnectionProfile later as some methods requires the ConnectionProfile instead of interface.